source/includes/changelogs/releases/100.9.0.rst

```diff
@@ -0,0 +1,14 @@
+.. _100.9.0-changelog:
+
+100.9.0 Changelog
+-----------------
+
+*Released 2023-10-20*
+
+This version adds support for MongoDB Server 7.0 and updates the Go version 
+used to build the tools to Go 1.20.10.
```

Comment:
Consider converting this to two separate bullets ([borrowed some verbiage from 100.7.5 for consistency](https://github.com/mongodb/docs-commandline-tools/blob/master/source/includes/changelogs/releases/100.7.5.rst)) :

```suggestion
- This version adds support for :manual:`MongoDB Server 7.0 </release-notes/7.0>`.
- This version updates the Go version used to build the tools `1.20.10<https://go.dev/doc/devel/release#go1.20.minor>`__ .
```